microsoft/utils/aitl/template.json

```diff
@@ -0,0 +1,39 @@
+{
+    "location": "westus3",
+    "properties": {
+        "vmSize": [
+            "Standard_D4_v4"
+        ],
+        "selections": [
+            {
+                "caseName": [],
+                "caseArea": [],
+                "caseCategory": [],
+                "casePriority": [
+                    0
+                ],
+                "caseTags": [],
+                "action": "include",
+                "testIterations": 1,
+                "retryCount": 0,
+                "useNewEnvironment": false,
+                "ignoreFailure": false
+            },
+            {
```

Comment:
Remove this part, it's confusing. One item is enough.

Reply:
I updated it to just one selection object.